src/llfs/data_packer.test.cpp

```diff
@@ -194,32 +194,36 @@ void run_pack_record_test(const usize count)
   struct MyTemp {
     int data;
   };
-  const u64 mem_size = 64;
-  std::array<u8, mem_size> memory;
-  memory.fill(0);
+  constexpr u64 kMemSize = 64;
+  u64 requested_space = sizeof(MyTemp) * count;
+  std::array<u8, kMemSize> memory;
 
-  std::array<u8, mem_size> expected;
-  expected.fill(0);
+  memory.fill(0);
 
   llfs::DataPacker packer{llfs::MutableBuffer{memory.data(), memory.size()}};
 
-  EXPECT_EQ(packer.space(), mem_size);
+  EXPECT_EQ(packer.space(), kMemSize);
 
-  // allocate space for first lot based on 'count'
+  MyTemp* p_buff;
   if (count != 1) {
-    [[maybe_unused]] auto p_buff = packer.template pack_record(batt::StaticType<MyTemp>{}, count);
+    p_buff = packer.template pack_record(batt::StaticType<MyTemp>{}, count);
```

Comment:
I don't think the `template` keyword is needed here... please remove if this is true (or lmk why the compiler needs it, I'd be interested to know! :-))

Reply:
I saw an existing usage in item_layout.hpp. In fact I had it without the 'template' keyword earlier.  Then, I thought I should just follow the existing usage. I can change it back.